.github/workflow/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
```

Comment:
why adding a master branch?

Reply:
This is how its set up from the explanations on mkdocs, likely copied it from my test example and did not realize it. Will remove it.

Reply:
Done